Mathlib/MeasureTheory/Measure/OpenPos.lean

```diff
@@ -251,3 +251,49 @@ theorem measure_closedBall_pos (x : X) {r : ℝ≥0∞} (hr : r ≠ 0) : 0 < μ
 #align emetric.measure_closed_ball_pos EMetric.measure_closedBall_pos
 
 end EMetric
+
+section MeasureZero
+/-! ## Meagre sets and measure zero
+In general, neither of meagre and measure zero implies the other.
+- for all $α ∈ (0,1)$, there is a generalised Cantor set $C ⊆ [0,1]$ of measure `α`.
+Cantor sets are nowhere dense. (Taking a countable union of fat Cantor sets whose
+measure approaches 1 yields a meagre set of measure 1.)
+- ℚ ⊆ ℝ has measure zero, but is dense (in particular, not meagre).
+
+However, **closed** measure zero sets are nowhere dense,
+and σ-compact measure zero sets in a Hausdorff space are meagre.
+-/
+open Function TopologicalSpace Set MeasureTheory Measure
+
+variable {X : Type*} [TopologicalSpace X] [MeasurableSpace X] [BorelSpace X]
+  {μ : Measure X} [IsOpenPosMeasure μ]
+
+lemma open_measure_zero_implies_empty {s : Set X} (hs : IsOpen s) (hs' : μ s = 0) : s = ∅ :=
```

Comment:
```suggestion
lemma eq_empty_of_isOpen_measure_zero {s : Set X} (hs : IsOpen s) (hs' : μ s = 0) : s = ∅ :=
```
The naming convention rather starts with the conclusion, and names relevant assumption after `of`. Since the property of being open is called `IsOpen`, this is what we should have in the name, not just `open`.